### PR TITLE
feat / prints timestamp for executed BUY/SELL orders with PMM strategy

### LINKED ISSUE #3097: Add timestamps to notification when a buy/sell order is filled
**Feature Suggestion**
So you know... here;
this
but with timestamps, delicious timestamps

**Impact**
Im not looking at the log all days, but when I do, I miss the feeling of knowing approx when in time things happened, was the spread of fulfilled orders... mostly in the beginning of the day? the end of the day? a bit here and there?

**Additional context**
Pretend the above screenshoot lives here but is visiting the Feature Suggestion as it likes that neighborhood since this was teh original place to out it according to the guidelines, but it is a bit mischievous

### PR DESCRIPTION
**Pull request by willzs03**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

Closes #3097 

timestamps
Added a feature that prints the timestamps of executed orders, as shown in the picture above. At the moment, it only works with the PMM strategy, as only the PMM file was modified. This could potentially work with other strategies, you'd just have to copy paste the necessary lines highlighted in pure_market_making.pyx.

I'm a total newbie using github with forks and pull requests. I did follow the Contributing.MD tutorial, but I don't think I set it up correctly. Ultimately, the major changes for the feature are in the PMM file, which is fairly straightforward. 

My 'inspiration' to work on this feature came from issue [3097](https://github.com/CoinAlpha/hummingbot/issues/3097), where OP wanted a timestamp feature for executed trades.

### CODE REVIEW

File: hummingbot/strategy/pure_market_making/pure_market_making.pyx [hunk 1]
```diff
@@ -1023,7 +1024,10 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             f"({limit_order_record.quantity} {limit_order_record.base_currency} @ "
             f"{limit_order_record.price} {limit_order_record.quote_currency}) has been completely filled."
         )
+        buyTimestamp = self._current_timestamp
+        buyDateTime = datetime.fromtimestamp(buyTimestamp)
```
Comment (keithbaum) [inline hummingbot/strategy/pure_market_making/pure_market_making.pyx:1028]: How about condensing these two lines (and removing import of datetime) by using:
pd.Timestamp.fromtimestamp(self._current_timestamp)

Please try it out and if successful I will approve this code

File: hummingbot/strategy/pure_market_making/pure_market_making.pyx [hunk 2]
```diff
@@ -1067,7 +1071,10 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             f"({limit_order_record.quantity} {limit_order_record.base_currency} @ "
             f"{limit_order_record.price} {limit_order_record.quote_currency}) has been completely filled."
         )
+        sellTimestamp = self._current_timestamp
+        sellDateTime = datetime.fromtimestamp(sellTimestamp)
```
Comment (keithbaum) [inline hummingbot/strategy/pure_market_making/pure_market_making.pyx:1075]: same applies here

**Comment by keithbaum** [comment]
@willzs03 Please check out my comments on the changes

**Comment by willzs03** [comment]
Hey @keithbaum, sorry, I'm away on vacation and I don't have a device on hand that has hummingbot setup. I will review and test the changes you mentioned when I get back, if they're still open by then. This should be around mid August.

**Comment by aarmoa** [comment]
Hello @willzs03. I have created a new PR that covers this change and extends it to be used in any strategy. The new PR is #3603. I will close this PR (I have already included a link to this in the new PR as the original idea)

**Pull request closed without merging**